scripts/check_links.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+excludeList=(
+    ".*maven.apache.org"
+    ".*docs.npmjs.com"
+    ".*github.com"
+    "www.php.net"
+    "aiven.io\\/community"
+    "aiven.io\\/build"
+    "aiven.io\\/docs\\/assets"
+    ".*uptime.aiven.io"
+    ".*linkedin.com"
+    ".*cdn.sanity.io"
+    ".*console.cloud.google.com"
+    "docker.com"
+  )
+
+rateLimit="--rate-limit=30"
+limitConnections="--max-connections=9"
+bufferSize="--buffer-size=8192"
+timeout="--timeout=4000"
+address="https://aiven.io/docs/"
+redirectionLimit="--max-redirections=5"
+acceptedResponses="--accepted-status-codes=200..404"
+
+muffet ${bufferSize} \
+       ${timeout} \
+       ${excludeList[@]/#/--exclude } \
+       ${skipTLS} \
+       ${excludeGithub} \
```

Comment:
`skipTLS` and `excludeGithub` are referenced but are not defined. Why not simply inline the options?

```bash
muffet --buffer-size=8192 \
       --timeout=4000 \
       "${excludeList[@]/#/--exclude }" \
       --max-connections=9 \
       --rate-limit=30 \
       --max-redirections=5 \
       --accepted-status-codes=200..404 \
       --color=always \
       https://aiven.io/docs/
```

Reply:
Thanks for spotting the things I forgot to cleanup 🙌  I'm not inlining params because I'm foreseeing a next update in which I run muffet on local build with different values, so already exporting params like this makes things a little easier for the next step :)